DEVEL.md

```diff
@@ -0,0 +1,68 @@
+# Developing the Maistra Istio Operator
+
```

Comment:
We should probably add one or two sentences on how to build the operator

Reply:
Added.